reactDashboard/src/components/DataCharts/DataCharts.css

```diff
@@ -58,43 +76,58 @@
 .region-chart-container {
   height: 100%;
   width: 100%;
-  background-color: #ffffff;
+  background-color: white;
 }
 
 .sonification-play-button {
   position: absolute;
   top: 20px;
   right: 40px;
-  background-color: #ffffff;
+  background-color: white;
   font-size: 0.5rem;
   border: 1px solid #6c6c6c;
   border-radius: 10px;
   padding: 7px;
   width: 7em;
   height: 7em;
+  color: black;
+}
+
+.sonification-play-button {
```

Comment:
Can this be merged with the block above and duplicate attributes removed please?

Reply:
✅